docs/Makefile

```diff
@@ -0,0 +1,60 @@
+BUILD_DIR = ../docs_build
+ROOTDIR = $(realpath .)
+NAME = watcher-operator
+DEST_DIR = $(BUILD_DIR)/$(NAME)
+DEST_HTML = $(DEST_DIR)/index.html
+DEST_PDF = $(BUILD_DIR)/$(NAME).pdf
+IMAGES_DIR = $(DEST_DIR)/images
+IMAGES_TS = $(DEST_DIR)/.timestamp-images
+MAIN_SOURCE = main.adoc
+OTHER_SOURCES = $(shell find ./assemblies -type f)
+IMAGES = $(shell find ./images -type f)
+ALL_SOURCES = $(MAIN_SOURCE) $(OTHER_SOURCES) $(IMAGES)
+UNAME = $(shell uname)
+BUNDLE_EXEC ?= bundle exec
+
+ifeq ($(UNAME), Linux)
+BROWSER_OPEN = xdg-open
+endif
+ifeq ($(UNAME), Darwin)
+BROWSER_OPEN = open
+endif
+
+all: html
+
+html: html-latest
+
+html-latest: prepare $(IMAGES_TS) $(DEST_HTML)
+
+pdf: prepare $(DEST_PDF)
+
+prepare:
+	@mkdir -p $(BUILD_DIR)
+	@mkdir -p $(DEST_DIR) $(IMAGES_DIR)
+
+clean:
+	@rm -rf "$(DEST_DIR)" "$(DEST_PDF)"
+
+watch-html:
+	@which inotifywait > /dev/null || ( echo "ERROR: inotifywait not found, install inotify-tools" && exit 1 )
+	while true; do \
+		inotifywait -r -e modify -e create -e delete .; \
+		sleep 0.5; \
+		$(MAKE) html; \
+	done
+
+open-html: html
+	${BROWSER_OPEN} "file://$(realpath $(ROOTDIR)/$(DEST_HTML))"
+
+open-pdf: pdf
```

Comment:
is this one redundant we aren't wiring it up anywhere

Reply:
Done!